### PR TITLE
11.0.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by tristantarrant**

### CODE REVIEW

**Comment by ryanemerson** [comment]
The bump seems to have broken the store migrator and the WFLY integration.

File: integrationtests/wildfly-modules/src/test/resources/infinispan-custom.xml [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+        xmlns:server="urn:infinispan:server:${infinispan.core.schema.version}">
 
     <cache-container default-cache="default" statistics="true">
         <transport cluster="${infinispan.cluster.name}" stack="${infinispan.cluster.stack:tcp}"/>
         <global-state />
         <local-cache name="default" />
     </cache-container>
 
-    <server xmlns="urn:infinispan:server:10.1">
+    <server xmlns="urn:infinispan:server:11.0">
```
Comment (danberindei) [inline integrationtests/wildfly-modules/src/test/resources/infinispan-custom.xml]: Shouldn't you be using `${infinispan.core.schema.version}`?

Comment (tristantarrant) [reply integrationtests/wildfly-modules/src/test/resources/infinispan-custom.xml]: Done

**Comment by tristantarrant** [comment]
Ok, CI looks good now

**Comment by danberindei** [comment]
Merged, thanks Tristan!

**Pull request merged**